src/coreclr/jit/gentree.cpp

```diff
@@ -24047,7 +24088,7 @@ void ReturnTypeDesc::InitializeStructReturnType(Compiler*                comp,
                 m_regType[i] = comp->getJitGCType(gcPtrs[i]);
             }
 
-#elif defined(TARGET_LOONGARCH64)
+#elif defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
             assert((structSize >= TARGET_POINTER_SIZE) && (structSize <= (2 * TARGET_POINTER_SIZE)));
 
             uint32_t floatFieldFlags = comp->info.compCompHnd->getLoongArch64PassStructInRegisterFlags(retClsHnd);
```

Comment:
Seems a bit odd to call `getLoongArch64PassStructInRegisterFlags` here, but I guess the ABIs are close enough/equivalent for this to work out?

Reply:
Sorry. It is really odd even if these are very close. I created a new for RISCV64 and pushed a commit in both vm and jit PRs.
Thank you.

Reply:
This also requires updating the JIT-EE GUID and ThunkInput.txt. See https://github.com/dotnet/runtime/blob/main/docs/project/updating-jitinterface.md for more details.

Reply:
I don't think it is ok to leave it unimplemented on the VM side in this PR. Can you introduce a skeleton for this new JIT-EE API? If you need some more help then #78593 is an example PR that introduces a new JIT-EE API.

Reply:
I already added ThunkInput.txt and the other codes into https://github.com/dotnet/runtime/pull/82381. (*. I missed GUID update.)

If you think that all related codes should be in a PR, then how about reverting the last commit and make a new PR after this coreclr-jit and coreclr-vm (https://github.com/dotnet/runtime/pull/82381) PR are merged?
If not, I will update GUID in https://github.com/dotnet/runtime/pull/82381 and so on.
Thank you

Reply:
The problem is that we cannot merge this PR without coherent JIT-EE implementations/consumptions for all our tooling. I would suggest adding the helper (including the VM side and the rest of the boilerplate) as part of this PR.

Reply:
I moved coreclr-vm commit to this and updated GUID.
Thank you.